WORKSPACE

```diff
@@ -316,6 +316,14 @@ php_gapic_repositories()
 # C#
 ##############################################################################
 
+# Required to access the C#-specific common resources config.
+http_archive(
+    name = "google_cloud_dotnet",
+    urls = ["https://github.com/googleapis/google-cloud-dotnet/archive/1b0afcc7b11b02aaea3456f822c8030d5e9f361c.zip"],
```

Comment:
Is it what Ads is using? I mean it will work, but it basically creates a circular dependency between the repositories. This repo is used to create `google-cloud-dotnet`, and it imports `google-cloud-dotnet` as a dependency at the same time. Does this whole import exists to import a single file (CommonResourcesConfig.json)? If yes, why is it located in google-cloud-dotnet repostory? (not in googleapis, or gapic-generator-csharp for example)?